rai/node/utility.hpp

```diff
@@ -98,6 +98,9 @@ bool fetch_object (T & object, boost::filesystem::path const & path_a, std::fstr
 	return error;
 }
 
+/**
+ * RAII wrapper for MDB_env
```

Comment:
Any particular reason for RAII instead of Rai? Also since you are adding the commenting is there any reason we dont start moving it to NANO? same with like 133

Reply:
Other than this, LGTM

Reply:
RAII here means Resource Acquisition Is Initialization. It's a C++ principle that describes what this wrapper does. It's not rai specific.

Reply:
http://en.cppreference.com/w/cpp/language/raii